### PR TITLE
Uai cmk

### PR DESCRIPTION
**Pull request by DeMale**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [x] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

*
*
*

### CODE REVIEW

**Comment by DeMale** [comment]
@bmoore-msft  could you please review this?

File: 201-machine-learning-advanced/azuredeploy.json [hunk 1]
```diff
@@ -298,6 +298,13 @@
         "description": "Specifies if the customer managed keyvault key uri. Required when encryption is enabled"
       }
     },
+    "user_assigned_identity_for_cmk_encryption": {
```
Comment (zhaomuzhi) [inline 201-machine-learning-advanced/azuredeploy.json:301]: I think it won't work without userAssigned identity type. Have you tested it?

File: 201-machine-learning-advanced/azuredeploy.json [hunk 2]
```diff
@@ -298,6 +298,13 @@
...
+      "type": "string",
+      "defaultValue": "",
```
Comment (bmoore-msft) [inline 201-machine-learning-advanced/azuredeploy.json:303]: if this works without it, why would I ever need to add it?

**Comment by bmoore-msft** [comment]
Closing abandoned request - please reopen if you can address the feedback.

**Pull request closed without merging**